homeassistant/components/mqtt/discovery.py

```diff
@@ -44,6 +53,12 @@
     }
 }
 
+IMPLICIT_STATE_COMPONENTS = [
```

Comment:
Can you add a comment why we treat some differently?

Reply:
Fixed.